LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 Angela Guette, Wiebke Frey, Laura Garcia
```

Comment:
Should be Homeday, not you personally

Reply:
Maybe also for the gemspec authors? Better ask mgmt what's right here 😉

Reply:
Homeday sounds fine ☺️